exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -71,13 +73,16 @@ func NewPrwExporter(namespace string, endpoint string, client *http.Client, exte
 		return nil, errors.New("invalid endpoint")
 	}
 
+	userAgentHeader := fmt.Sprintf("%s/%s", strings.ReplaceAll(strings.ToLower(buildInfo.Description), " ", "-"), buildInfo.Version)
```

Comment:
This seems like an arbitrary sanitization rule. Description can include any unicode character. We should sanitize it all if we want to autogenerate something from the description. Another alternative wouild be adding a new field to BuildInfo to allow setting custom user agents.

Also don't lowercase, uppercase is valid user agent. See https://www.ietf.org/rfc/rfc2616.html#section-3.8